examples/http-ts/src/queryToStore.ts

```diff
@@ -0,0 +1,24 @@
+import { URLSearchParams } from 'url';
+import * as store from '@leapfrogtechnology/async-store';
+
+/**
+ * Get the query params from the url and set them to async-store with `query` key.
+ *
+ * @param url The url to parse query params from.
+ */
+export default function(urlQueryParams: URLSearchParams): void {
```

Comment:
Have a concise name for the indentifier
```suggestion
export default function(params: URLSearchParams): void {
```